docs/cimpl6LanguageReference.md

```diff
@@ -750,7 +740,7 @@ Group: QuestionWithTemporalAnswer
 Parent: QuestionAndAnswer
 Description: "A question whose answer is a date or a time"
 Answer substitute TemporalAnswer  
-// _Do not_ use one of these patterns "Answer only date or time" or "Answer only TemporalAnswer"
+// DO NOT use one of these patterns: "Answer only date or time" or "Answer only TemporalAnswer"
```

Comment:
DO NOT should be italicized, not capitalized.  Emphasis is italics.

Reply:
I left it unemphasized (lower case). Changing the entire block to <pre> changes the format too much.